crates/common/consensus/src/deneb/beacon_state.rs

```diff
@@ -1225,20 +1229,21 @@ impl BeaconState {
         let old_previous_justified_checkpoint = self.previous_justified_checkpoint;
         let old_current_justified_checkpoint = self.current_justified_checkpoint;
 
-        // Process justifications
         self.previous_justified_checkpoint = self.current_justified_checkpoint;
-        for i in 1..JUSTIFICATION_BITS_LENGTH as usize {
+
+        for i in (1..JUSTIFICATION_BITS_LENGTH).rev() {
```

Comment:
```suggestion
        for i in (1..JUSTIFICATION_BITS_LENGTH).rev().map(|i| i as usize) {
```

Nit, but this can make duplicated `usize` casting